validator/schema/image_schema.py

```diff
@@ -191,6 +191,7 @@ def image_schema(file):
             'manifests-dir': And(str, len),
             'bundle-dir': And(str, len),
             'registry': And(str, len),
+            'valid-subscription-label': str,
```

Comment:
Think this should not be an empty string, hence `And(str, len)`

Reply:
done 👍🏾